vendor.conf

```diff
@@ -159,7 +159,7 @@ github.com/morikuni/aec 39771216ff4c63d11f5e604076f9c45e8be1067b
 github.com/docker/go-metrics d466d4f6fd960e01820085bd7e1a24426ee7ef18
 
 # stacks and their deps
-github.com/docker/stacks 8e8ca77c0ac58e3f2fafbf7490340be1a91c8659 git@github.com:docker/stacks.git
+github.com/docker/stacks 68ec0f837572a4e63e4e8993c8b3504a99eb9c2e /home/dperny/go/src/github.com/docker/stacks
```

Comment:
repo is public now, so `/home/dperny/go/src/github.com/docker/stacks` could be removed (I guess `vndr` is failing on that right now)

Reply:
oof yeah